### PR TITLE
Add husky

### PR DESCRIPTION
**Pull request by turbaszek**
Depends on #71

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 ![Corona-games](https://github.com/mavend/corona-games/workflows/Corona-games/badge.svg)
 [![Netlify Status](https://api.netlify.com/api/v1/badges/866fc714-bea9-443b-920b-3d50c022710e/deploy-status)](https://app.netlify.com/sites/corona-games/deploys)
+[![code style: prettier](https://img.shields.io/badge/code_style-prettier-ff69b4.svg?style=flat-square)](https://github.com/prettier/prettier)
```
Comment (pastDexter) [inline README.md:3]: najważniejsze! 🎉

File: package.json [hunk 2]
```diff
@@ -61,5 +63,10 @@
     "@storybook/preset-create-react-app": "^2.1.1",
     "@storybook/react": "^5.3.17",
     "prettier": "^2.0.2"
+  },
+  "husky": {
+    "hooks": {
+      "pre-commit": "npm run prettier"
```
Comment (Bombasarkadian) [inline package.json:69]: I'm too late as it's already been merged, but should we run a command that changes files in a `pre-commit`? Shouldn't it only run checks? @turbaszek @pastDexter

Comment (Bombasarkadian) [reply package.json:69]: Yup. Running this command doesn't prevent you from making a commit. All files are committed with wrong formatting and corrected files are left unstaged

Comment (Bombasarkadian) [reply package.json:69]: https://prettier.io/docs/en/precommit.html#option-2-pretty-quickhttpsgithubcomazzpretty-quick ?

Comment (pastDexter) [reply package.json:69]: :O you are right

Comment (turbaszek) [reply package.json:69]: Too much moving parts... does it mean that we need install `pretty-quick`?

Comment (turbaszek) [reply package.json:69]: It's quite interesting that prettier doesn't rise error when files were staged

Comment (Bombasarkadian) [reply package.json:69]: `pretty-quick` seems to handle adding updated files to the commit and it only runs on changed files, which should be much faster than it is now

Comment (Bombasarkadian) [reply package.json:69]: Why would it raise any error? It's a perfectly valid situation

Comment (turbaszek) [reply package.json:69]: All git commits hooks I know (and we have over 20 in Airflow) fails when they change file.

**Pull request merged**